webui/tests/cypress/integration/02-experiments.spec.ts

```diff
@@ -1,47 +1,44 @@
-describe('experiment List', () => {
-  const recordSelector = '#experimentsList tr.record';
+describe('Experiment List', () => {
+  const recordSelector = 'tr.ant-table-row';
 
   before(() => {
     cy.login();
-    cy.visit('/ui/experiments');
+    cy.visit('/det/experiments');
   });
 
   describe('batch buttons', () => {
-    it('should have 4 buttons', () => {
-      cy.get('.batchActions button').should('have.lengthOf', 4).should('be.disabled');
-    });
-
-    it('should be disabled at start', () => {
-      cy.get('.batchActions button').should('be.disabled');
+    it('should have 7 buttons', () => {
+      cy.get('thead input[type=checkbox]').click();
+      cy.get('[class*="TableBatch_actions_"] button').should('have.lengthOf', 7);
     });
 
     describe('Open TensorBoard', () => {
       it('should be enabled when all experiments are selected', () => {
-        cy.get('.batchActions input[type=checkbox]').click();
-        cy.get('.batchActions input[type=checkbox]')
-          .should('have.lengthOf', 1).should('be.enabled');
-        cy.get('.batchActions button').contains(/tensorboard/i)
-          .closest('button').should('be.enabled');
-        cy.get('.batchActions input[type=checkbox]').click();
+        cy.get('thead input[type=checkbox]').click();
+        cy.get('[class*="TableBatch_actions_"] button:first-child')
+          .should('contain', 'Open TensorBoard')
+          .should('be.enabled');
+        cy.get('thead input[type=checkbox]').click();
       });
     });
   });
 
   describe('table filter', () => {
     describe('archive toggle', () => {
-      it('should show archived experiments when clicked', () => {
+      it('should show and hide archived experiments', () => {
         cy.get(recordSelector).should('have.length', 3);
-        cy.get('#experimentsList .filters input[type=checkbox]').click();
+        cy.get('[class*="Toggle_base_"] button').click();
         cy.get(recordSelector).should('have.length', 4);
-        cy.get('#experimentsList .filters input[type=checkbox]').click();
+        cy.get('[class*="Toggle_base_"] button').click();
         cy.get(recordSelector).should('have.length', 3);
       });
 
-      it('should default to hiding archived experiments', () => {
-        cy.get('#experimentsList .filters input[type=checkbox]').should('not.have.attr', 'checked');
-        cy.get(recordSelector).should('have.length', 3);
-        cy.get(recordSelector).should('not.contain', 'Yes');
-      });
+      // TODO: fix when Archived column has landed
```

Comment:
blocking: let's skip instead of commenting it out.

Reply:
@hkang1 I think now that you're skipping the whole `describe` block we wouldn't need to comment the test below?

Reply:
https://github.com/determined-ai/determined/pull/992/files?file-filters%5B%5D=.ts&hide-deleted-files=true#diff-41e695829ae8493a32277a25cd0eb9c0R30 or it still gets called?

Reply:
actually isn't that skip skipping the test that you updated?

Reply:
forgot to re-enable it while testing other parts. could probably use a lint rule to check for `skips` :D